action.yml

```diff
@@ -0,0 +1,64 @@
+name: Replay Cypress
+author: Replay.io
+description: Run and record Cypress tests with Replay
+inputs:
+  apiKey:
+    required: true
+    description: "Replay.io API Key"
+  browser:
+    description: "Replay Browser to use"
+    default: Replay Firefox
+  command:
+    description: "Command to run tests"
+    default: npx cypress run
+  issue-number:
+    description: "Pull Request on which to comment results"
+  public:
+    description: "Makes the uploaded replay viewable by everyone"
+    default: false
+  upload-all:
+    description: "Upload all recordings instead of only recordings of failed tests"
+    default: false
+  working-directory:
+    description: "Relative path to app"
+    default: "."
+runs:
+  using: composite
+  steps:
+    - name: Generate Test Run ID
+      id: test-run-id
+      run: echo "::set-output name=UUID::$(uuidgen | tr A-Z a-z)"
+      shell: bash
+    - name: Generate metadata file
+      id: metadata-file
+      run: echo "::set-output name=TMP_FILE::$(mktemp)"
+      shell: bash
+    - run: npm i --legacy-peer-deps @replayio/cypress@">=0.2.0"
+      shell: sh
+    - run: ${{ inputs.command }} --browser "${{ inputs.browser }}"
+      shell: sh
+      working-directory: ${{ inputs.working-directory }}
+      env:
+        RECORD_REPLAY_TEST_RUN_ID: ${{ steps.test-run-id.outputs.UUID }}
+        RECORD_ALL_CONTENT: 1
+        RECORD_REPLAY_METADATA_FILE: ${{ steps.metadata-file.outputs.TMP_FILE }}
+        CYPRESS_NO_COMMAND_LOG: 1
```

Comment:
Should this be an option? I think the Command Log actually makes it easier to debug the tests.

Reply:
It doesn't work right now (in v10) so we can remove it. The idea was to exclude it and produce a similar experience in Replay itself (rather than the recording). That's a bit further out though so probably better to keep it (once it's supported).